src/test/java/seedu/address/logic/parser/CovigentAppParserTest.java

```diff
@@ -65,6 +67,19 @@ public void parseCommand_editPatient() throws Exception {
         assertEquals(new EditPatientCommand(patient.getName().toString(), descriptor), command);
     }
 
+    @Test
+    public void parseCommand_editTask() throws Exception {
+        Task task = new TaskBuilder().build();
+
+        EditTaskDescriptor descriptor = new EditTaskDescriptor();
+        descriptor.setDescription(task.getDescription());
+        descriptor.setDateTimeDue(task.getDueAt());
+
+        EditTaskCommand command = (EditTaskCommand) parser.parseCommand(
+                TaskUtil.getEditTaskCommand(task, VALID_ROOM_INDEX_ONE, VALID_TASK_INDEX_ONE));
+        assertEquals(new EditTaskCommand(VALID_ROOM_INDEX_ONE, VALID_TASK_INDEX_ONE, descriptor), command);
+    }
+
     @Test
     public void parseCommand_deletePatient() throws Exception {
```

Comment:
Can rename it into parseCommand_deletePatient_success() instead?

Reply:
Yes, @LeeMingDe mentioned this too!